### PR TITLE
Fix cross-references in API changes < 2.0.0.

### PR DESCRIPTION
**Pull request by QuLogic**
## PR Summary

More of the same. No backport since it changes references to code-style for deleted targets.

## PR Checklist

- [N/A] Has Pytest style unit tests
- [N/A] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [N/A] New features are documented, with examples if plot related
- [x] Documentation is sphinx and numpydoc compliant
- [N/A] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [N/A] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: doc/api/prev_api_changes/api_changes_1.4.x.rst [hunk 1]
```diff
@@ -79,112 +80,115 @@ original location:
...
-    - `xyann` -> set the location of the annotation
-    - `xy` -> set where the arrow points to
-    - `anncoords` -> set the units of the annotation location
-    - `xycoords` -> set the units of the point location
-    - `set_position()` -> `Annotation` only set location of annotation
+    - *xyann* -> set the location of the annotation
+    - *xy* -> set where the arrow points to
+    - *anncoords* -> set the units of the annotation location
+    - *xycoords* -> set the units of the point location
+    - ``set_position()`` -> `.Annotation` only set location of annotation
```
Comment (anntzer) [inline doc/api/prev_api_changes/api_changes_1.4.x.rst]: '.Annotation.set_position'? (quotes -> backticks)

File: doc/api/prev_api_changes/api_changes_1.5.0.rst [hunk 2]
```diff
@@ -34,48 +34,48 @@ before that gets overhauled).
...
-Tightened validation so that only {'tip', 'tail', 'mid', and 'middle'}
-(but any capitalization) are valid values for the 'pivot' kwarg in
-the `Quiver.__init__` (and hence `Axes.quiver` and
-`plt.quiver` which both fully delegate to `Quiver`).  Previously any
-input matching 'mid.*' would be interpreted as 'middle', 'tip.*' as
-'tip' and any string not matching one of those patterns as 'tail'.
+Tightened validation so that only {'tip', 'tail', 'mid', and 'middle'} (but any
+capitalization) are valid values for the *pivot* keyword argument in the
+`.Quiver` class (and hence `.axes.Axes.quiver` and `.pyplot.quiver` which both
+fully delegate to `.Quiver`).  Previously any input matching 'mid.*' would be
```
Comment (anntzer) [inline doc/api/prev_api_changes/api_changes_1.5.0.rst]: needs quoting to avoid asterisk being interpreted as italic (just below too)

Comment (QuLogic) [reply doc/api/prev_api_changes/api_changes_1.5.0.rst]: That's only true on GitHub; it's correct in [the rendered result](https://24353-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/api/prev_api_changes/api_changes_1.5.0.html#tightened-input-validation-on-pivot-kwarg-to-quiver).

File: doc/api/prev_api_changes/api_changes_1.4.x.rst (hunk 1 above)
Comment (QuLogic) [reply doc/api/prev_api_changes/api_changes_1.4.x.rst]: There is no `.Annotation.set_position`; it's on the underlying `Text`. But this is referring to the Annotation, and the fact that `.set_position` now works on it correctly, so I'm not sure we want to redirect to `Text`.

**Pull request merged**